src/utils/containers.ts

```diff
@@ -1,96 +1,136 @@
-import tar from 'tar-fs';
-import Docker from 'dockerode';
+import tar from "tar-fs";
+import Docker from "dockerode";
+import * as stream from "stream";
 
-import HermitOptions from './lib/HermitOptions';
-import Syscall from './lib/Syscall';
-import logger from './logger';
-import { getNodeMajorVersion } from 'typescript';
+import HermitOptions from "./lib/HermitOptions";
+import Syscall from "./lib/Syscall";
+import logger from "./logger";
+import { getNodeMajorVersion } from "typescript";
 
 const docker = new Docker();
 
-export const buildImage = (options: HermitOptions) => new Promise<string>((resolve, reject) => {
-  const imageTag = Math.random().toString(36).substr(2, 3)
-    + Math.random().toString(36).substr(2, 3)
-    + Math.random().toString(36).substr(2, 4);
+export const runCommandInContainer = async (
+  image: string,
+  command: string
+): Promise<string> => {
+  await docker.pull(image);
 
-  const tarDir = tar.pack(options.path);
+  let output = "";
 
-  docker.buildImage(tarDir, { t: imageTag, dockerfile: 'Dockerfile.strace', q: true, forcerm: true }, (error, stream) => {
-    if (error) {
-      reject(error);
-    }
+  const writable = new stream.Writable({
```

Comment:
Interesting approach. Is the `Writible Stream` more stable than the previous approach?

Reply:
This was just to be able to get the output into a string instead of sending it to stdout for example